src/frontend/js/index.tsx

```diff
@@ -106,4 +106,4 @@ async function render() {
 document.addEventListener('DOMContentLoaded', render);
 
 // In some case, you would like to render Richie components manually
-(window as any).__RICHIE__ = Object.create({ render });
+window.__RICHIE__ = Object.create({ render });
```

Comment:
`__RICHIE__` attribute name does'nt reflects what it contains, maybe something like `__PRERENDERED_COMPONENT__` would be more meaningful ?

Reply:
Agree with you that `__RICHIE__` is unclear about it's content.
But I think this name should be kept as generic as possible to put all other stuff related to richie frontend  in the future. Furthermore, I think it's a good practice to keep `RICHIE` in the name as this property is mix with all other window properties, in this way we are able to distinguish richie's properties easily.

Reply:
I see your point :+1: